qiskit/circuit/library/standard_gates/x.py

```diff
@@ -1087,7 +1127,15 @@ def __init__(
         )
 
     def inverse(self, annotated: bool = False):
-        """Invert this gate. The MCX is its own inverse."""
+        """Invert this gate. The MCX is its own inverse.
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```